samples/client/petstore/java/apache-httpclient/docs/FakeApi.md

```diff
@@ -586,7 +586,7 @@ public class Example {
         Float _float = 3.4F; // Float | None
         String string = "string_example"; // String | None
         File binary = new File("/path/to/file"); // File | None
-        LocalDate date = LocalDate.now(); // LocalDate | None
+        LocalDate date = new LocalDate(); // LocalDate | None
```

Comment:
I think we don't need commit code change for the sample of other languages.

Reply:
I just regenerated the docs and this was the result.

Reply:
Change reverted and updated the required file only